src/packages/members/members/components/input-member/input-member.element.ts

```diff
@@ -143,16 +155,29 @@ export class UmbInputMemberElement extends FormControlMixin(UmbLitElement) {
 			<uui-ref-node name=${ifDefined(item.name)} detail=${ifDefined(item.id)}>
 				<!-- TODO: implement is deleted <uui-tag size="s" slot="tag" color="danger">Deleted</uui-tag> -->
 				<uui-action-bar slot="actions">
+					${this.#renderOpenButton(item)}
 					<uui-button
 						@click=${() => this._requestRemoveItem(item)}
-						label="Remove member ${item.name}"
-						>Remove</uui-button
-					>
+						label="Remove member ${item.name}">
+						Remove
```

Comment:
would you add localization here, please?

Reply:
Localized the label in commit 7e453b3d92ce23776f6d98247a12e9233f9a45d9.